config/boot.rb

```diff
@@ -1,3 +1,5 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+
+require 'bootsnap/setup' unless ENV["RAILS_ENV"] == "production" #Speed up boot time by caching expensive operations.
```

Comment:
```suggestion
require 'bootsnap/setup' unless ENV['RAILS_ENV'] == 'production' # Speed up boot time by caching expensive operations.
```

Sorry, couldn't resist!

Reply:
I like double quotes :) Changing between double and single when interpolation is added or removed seems to me unnecessary

Reply:
Well, It's for consistency... We don't have much of it within the whole project (while I think we should), but at least I'd prefer not to mix both styles within the same file. This file already uses single quotes, so I suggested changing it here also for consistency (within the file, and also within the line!).

Reply:
This is unacceptable, but accepted.